### ISSUE TITLE
Rails 5 compatibility

### ISSUE THREAD

**Issue by adambedford** [dvandersluis/compendium#13]
The version on RubyGems seems to be compatible with Rails 5, however the master branch of the repo specifies `< 4`.

Is there a reason for this? Bundler has installed `1.1.3.4` in my Rails 5 app, which doesn't seem to have the `table` report DSL.

**Comment by dvandersluis** [comment]
I believe there are some features that use Rails 3 so that's why I added the specification. I haven't tried it out under rails 5. 

If you want clone and override the version requirement, I'm curious if a) tests all pass under rails 5 (they probably should because I don't think I'm testing anything specifically railsy); and b) if there are any actual failures when running under rails 5.

Please feel free to create a PR with new tests / fixes or at least create issues if something breaks under rails 5 and then we can see about changing the requirements!

**Comment by cimtico** [comment]
It seems rails 5.1 removed alias_method_chain, which is being used here: https://github.com/dvandersluis/compendium/blob/547f890d412fa491e62987ab5cf1369358208b3e/config/initializers/rails/active_record/connection_adapters/quoting.rb#L13
Would it be possible to have a conditional use of this method?

Great gem by the way!

**Comment by dvandersluis** [comment]
@cimtico yeah, I'm going to drop support for Ruby < 2.2 so relying on `prepend` instead of `alias_method_chain` will be possible. I'll make this change shortly.

Additionally, if anyone wants to make PRs for Rails 4/5 support I'd appreciate that.
